app/2.0/docs/tools/polymer-json.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "`polymer.json` specification"
+---
+
+<!-- toc -->
+
+# About `polymer.json`
+
+Creating a `polymer.json` in your project directory allows you to store information about your project structure and desired build configuration(s). It is used by the [Polymer CLI](sdffsdkjhk) as a way to understand the structure of your application.
+
+To make sure your application builds correctly, create a `polymer.json` file at the top level of your project:
+
+```
+-shop
+  |-README.md
+  |-app.yaml
+  |-bower.json
+  |-build
+  |-data
+  |-images
+  |-index.html
+  |-manifest.json
+  |-polymer.json
+  |-service-worker.js
+  |-src
+  |-sw-precache-config.js
+  |-test
+```
+
+#Example `polymer.json` file
+
+Here’s an example polymer.json file from the [Shop app](https://github.com/Polymer/shop):
+
+`polymer.json`
+{.caption}
+```json
+{
+  "entrypoint": "index.html",
+  "shell": "src/shop-app.html",
+  "fragments": [
+    "src/shop-list.html",
+    "src/shop-detail.html",
+    "src/shop-cart.html",
+    "src/shop-checkout.html",
+    "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
+   "src/**/*",
+   "data/**/*",
+   "images/**/*",
+   "bower.json"
+  ],
+  "extraDependencies": [
+    "manifest.json",
+    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
+  ]
```

Comment:
Consider adding this to the end of the example polymer.json, as the new linter will need it, and it won't cause problems with the existing linter:

```json
],
"lint": {
  "rules": ["polymer-2-hybrid"]
}
```